src/features/unified-share-modal/EmailForm.js

```diff
@@ -48,9 +52,10 @@ type Props = {
     onSubmit: Function,
     openInviteCollaboratorsSection?: Function,
     recommendedSharingTooltipCalloutName?: ?string,
+    restrictedExternalEmails: Array<string>,
     selectedContacts: Array<Contact>,
     sendButtonProps?: Object,
-    shouldInvalidateExternalCollabs?: boolean,
```

Comment:
This is now implied with the existence of `restrictedExternalEmails`, so no longer needed as a separate prop.